gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip
```

Comment:
This doesn't seem right.  Why did the distribution type change from all to bin?

Reply:
'bin' just contains the runtime but no sample code or documentation. It's a smaller file and the type used for allwpilib. If we want to stick with 'all' I can switch back.

Reply:
I am ok switching.